src/Illuminate/Bus/Dispatcher.php

```diff
@@ -112,8 +112,9 @@ public function hasCommandHandler($command)
     /**
      * Retrieve the handler for a command.
      *
-     * @param  mixed  $command
+     * @param  mixed $command
```

Comment:
You're going to need to revert all these param space changes. There should be 2 spaces before and after the param type.